src/test/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionsControllerIntegrationTest.kt

```diff
@@ -24,7 +23,7 @@ class IdentificationSessionsControllerIntegrationTest(
     val attributes = listOf("DG1", "DG2")
 
     @Test
-    fun `requesting identification sessions returns 401 when the request is made without authentication`() {
+    fun `requesting identification sessions returns 401 when the request is made without basic authentication`() {
```

Comment:
Cleanup. ;) Probably a result of main merged in.

Reply:
yes again, the huge mess from the merge conflict 😓